docs/gathered-data.md

```diff
@@ -771,8 +771,16 @@ None
 
 ## HelmInfo
 
-Collects summarized info about the helm usage on a cluster
-in a generic fashion
+Collects summarized information about Helm usage on a Kubernetes cluster
+in a generic fashion. It filters resources by the label `app.kubernetes.io/managed-by=Helm`
+and ensures the presence of a valid `helm.sh/chart`. The data is then grouped by namespace.
```

Comment:
It is not clear (at least to me) that only sources with both labels are collected.